lib/model/boards.dart

```diff
@@ -14,7 +14,32 @@ final boardTemplates = <String, Board>{
     [CellState.empty, CellState.empty, CellState.empty],
     [CellState.empty, CellState.empty, CellState.empty],
     [CellState.empty, CellState.empty, CellState.outside],
-  ]
+  ],
+  'Pyramid': [
+    [CellState.outside, CellState.outside, CellState.empty, CellState.outside, CellState.outside],
+    [CellState.outside, CellState.empty, CellState.empty, CellState.empty, CellState.outside],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+  ],
+  '5 X 5': [
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+  ],
+  'Four in a row': [
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
```

Comment:
Why such name?

Reply:
The first player to get four-in-a-row vertically, horizontally, or diagonally wins (instead of classic three)